services/indexes/avm/db_getters.go

```diff
@@ -266,15 +266,19 @@ func (db *DB) ListTransactions(ctx context.Context, p *ListTransactionsParams) (
 		return nil, err
 	}
 
-	count := uint64(p.Offset) + uint64(len(txs))
-	if len(txs) >= p.Limit {
-		p.ListParams = params.ListParams{}
-		err := p.Apply(dbRunner.
-			Select("COUNT(avm_transactions.id)").
-			From("avm_transactions")).
-			LoadOneContext(ctx, &count)
-		if err != nil {
-			return nil, err
+	var count uint64
+	count = 0
```

Comment:
This line is unnecessary in Go because all variables are guaranteed to start off in the zero value which is 0 for integer types.